src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -231,9 +288,26 @@ def download_genome_files(gene_ids: list[str], target_file_ext: list[str], exclu
     return failed_ids
 
 
+def create_softlink_between_csd_and_work_dir(
```

Comment:
besides genome files, I think we also need to link the taxonomy file downloaded from `_fetch_gtdb_genome_ids`. Those files are used by taxa count parser. Unless we want to reference to the sourcedata directory when we run taxa count parser.

Reply:
Updated!